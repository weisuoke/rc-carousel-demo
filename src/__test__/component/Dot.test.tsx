import { configure, render } from 'enzyme'
import * as Adapter from 'enzyme-adapter-react-16'
import toJson from 'enzyme-to-json'
import 'jest-styled-components'
import * as React from 'react'
import IndicatorDot from '../../component/Dot'

configure({ adapter: new Adapter() })

describe('it works', () => {
  it('Dots', () => {
    const wrapper = render(<IndicatorDot total={5} currentIndex={1} width={200} selesctedColor={'red'} />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
