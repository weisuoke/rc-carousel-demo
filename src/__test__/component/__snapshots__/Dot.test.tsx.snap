// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works Dots 1`] = `
.c1 {
  display: inline-block width:0.3rem height:0.3rem border-radius:0.15rem background-color:red margin:0.3rem 0.15rem opacity:1 transition-duration:0.3s;
}

.c2 {
  display: inline-block width:0.3rem height:0.3rem border-radius:0.15rem background-color:grey margin:0.3rem 0.15rem opacity:0.3 transition-duration:0.3s;
}

.c0 {
  position: absolute width:200px z-index:100 bottom:0.1rem text-align:center background-color:transparent;
}

<div
  class="c0"
  width="200"
>
  <span
    class="c1"
    selected=""
  />
  <span
    class="c2"
  />
  <span
    class="c2"
  />
</div>
`;
